src/api/todos.ts

```diff
@@ -0,0 +1,20 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const USER_ID = 391;
+
+export const getTodos = () => {
+  return client.get<Todo[]>(`/todos?userId=${USER_ID}`);
+};
+
+export const deleteTodos = (todoId: number) => {
+  return client.delete(`/todos/${todoId}`);
+};
+
+export const createTodos = ({ title, userId, completed }: Todo) => {
+  return client.post<Todo>(`/todos`, { title, userId, completed });
+};
```

Comment:
You can simplify it: 
```suggestion
export const createTodos = (data: Todo) => {
  return client.post<Todo>(`/todos`, data);
};
```

Reply:
> You can simplify it:

Good point

Reply:
I use id to control the focus of the input in the FormTodo component. To avoid creating a new state, I use this state:  
const [processingIds, setProcessingIds] = useState<number[]>([]);

Reply:
I can't use this option (because the object contains an id)